addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4652,6 +4652,16 @@ export class OdooEditor extends EventTarget {
         for (const el of element.querySelectorAll('*[class=""]')) {
             el.removeAttribute('class');
         }
+
+        // change t-field inline nodes to div
+        for (const el of element.querySelectorAll('b[data-oe-field][data-oe-type="html"]')) {
+            const blockEl = this.document.createElement('div');
+            for (const attr of el.attributes) {
+                blockEl.setAttribute(attr.name, attr.value);
+            }
+            blockEl.innerHTML = el.innerHTML;
+            el.parentNode.replaceChild(blockEl, el);
```

Comment:
You can do `el.replaceWith(blockEl);`. Doesn't change anything but it's just smoother.